shared/build.gradle

```diff
@@ -56,6 +70,12 @@ kotlin {
     }
 }
 
+dependencies {
+    kapt 'io.realm:realm-annotations-processor:5.14.0'
+
+    implementation 'com.squareup.retrofit2:converter-gson:2.5.0'
+}
+
 // workaround for https://youtrack.jetbrains.com/issue/KT-27170
 configurations {
     compileClasspath
```

Comment:
If someone more experienced with Gradle could look at this closely I would appreciate it!